### DISCUSSION TITLE
Whether there is any extention to display how many times a file is referenced?

### DISCUSSION THREAD

**Discussion by liaodalin19903** [microsoft/vscode-discussions#1211] [Extension Development QnA] [answered]
![C62B47A7-921C-4717-BF5D-6CBBBA43FF53](https://github.com/microsoft/vscode-discussions/assets/12727582/c1beb399-c23a-4583-a715-69e2fca92528)

Such as in the snapshot, `import {db} from db.ts`  , there will display the number of it was referered.

**Comment by isidorn** [comment, marked as answer]
Such an extension does not exist - at least to my knowledge.
Feel free to go ahead and build something like this.
